index.html

```diff
@@ -31,7 +31,9 @@
 <body class="page page-home">
     <section class="intro">
         <p>Welcome to Python Delhi User Group</p>
-        <h3>PyDelhi</h3>
+        <div itemscope itemtype="http://schema.org/Organization">
```

Comment:
@ntavish let's add this to `<html>` itself so you don't have to create another un-semantic `div` :)